src/Microsoft.DotNet.Build.Tasks.Packaging/src/GenerateRuntimeGraph.cs

```diff
@@ -140,6 +195,18 @@ public override bool Execute()
             return !Log.HasLoggedErrors;
         }
 
+        private void EnsureWritable(string file)
+        {
+            if (File.Exists(file))
+            {
+                var existingAttributes = File.GetAttributes(file);
+
+                if ((existingAttributes & FileAttributes.ReadOnly) != 0)
+                {
+                    File.SetAttributes(file, existingAttributes &= ~FileAttributes.ReadOnly);
```

Comment:
Cannot you still force overwrite it without changing the file attributes if it is readonly?

Reply:
How would you do that?  CreateFile will fail when opening read only file for write.  It will also fail when CREATE_ALWAYS is specified.  I could move/copy but that's more work than just clearing the bit. I've tested this and it does the right thing.

Reply:
OK. I'm fine with this just thought there was a force overwrite option.